src/components/UTIL_PageHeader.component

```diff
@@ -1,68 +1,73 @@
     <apex:attribute name="header" type="String" default=""
-        description="Subtitle under breadcrumb title."/>
+                    description="Subtitle under breadcrumb title."/>
     <apex:attribute name="icon" type="String" default=""
-        description="The name of the icon .png to display in the header, e.g. account, custom1"/>
+                    description="The name of the icon .png to display in the header, e.g. account, custom1"/>
     <apex:attribute name="iconCategory" type="String" default=""
-        description="The category the icon belongs to, e.g. standard, utility"/>
+                    description="The category the icon belongs to, e.g. standard, utility"/>
     <apex:attribute name="iconContainer" type="String"
-        description="The icon name for the container class will default to the icon name, e.g. account, custom1"/>
+                    description="The icon name for the container class will default to the icon name, e.g. account, custom1"/>
     <apex:attribute name="useNPSPicon" type="Boolean" default="false"
-        description="Use the RGB_npsp-stacked-color.png icon from static resources in the header instead of lex icon"/>
+                    description="Use the RGB_npsp-stacked-color.png icon from static resources in the header instead of lex icon"/>
     <apex:attribute name="smallSubtitleFont" type="Boolean" default="false"
                     description="When true, longer text in the header subtitle text is displayed in a smaller font"/>
     <!-- Cancel button attributes -->
     <apex:attribute name="showCancelBtn" type="Boolean" default="true"
-        description="Set to false to hide cancel button, default is true"/>
+                    description="Set to false to hide cancel button, default is true"/>
     <apex:attribute name="cancelAction" type="ApexPages.Action" default=""
-        description="Action for cancel button."/>
+                    description="Action for cancel button."/>
     <apex:attribute name="cancelLabel" type="String"
-        description="Label for cancel button. When blank will display stgBtnCancel label."/>
+                    description="Label for cancel button. When blank will display stgBtnCancel label."/>
     <apex:attribute name="cancelImmediate" type="Boolean" default="true"
-        description="Whether the cancel action should be immediate, defaults to true."/>
+                    description="Whether the cancel action should be immediate, defaults to true."/>
     <apex:attribute name="cancelDisabled" type="Boolean" default="false"
-        description="Whether the cancel button should be disabled, defaults to false."/>
+                    description="Whether the cancel button should be disabled, defaults to false."/>
+    <apex:attribute name="cancelDisabledOnClick" type="Boolean" default="false"
+                    description="Whether the cancel button should be disabled when clicked/pressed.  Must be rerendered to enable again."/>
```

Comment:
Can you update the descriptions for these to include "Defaults to false" to match the other attributes?

Reply:
Done.